email-sending-concepts/dmarc.md

```diff
@@ -0,0 +1,44 @@
+# DMARC
+
+**DMARC** stands for **Domain-based Message Authentication, Reporting and Conformance**.
+
+It is an email authentication protocol that builds on SPF and DKIM. DMARC allows domain owners to:
+
+- Tell receiving mail servers how to handle unauthenticated mail claiming to be from their domain
+- Receive reports about who is sending emails using their domain (legitimately or not)
+
+---
+
+## 🔍 How DMARC Works
+
+1. A mail server receives an email claiming to be from `example.com`
+2. It checks whether the email passes SPF **or** DKIM
+3. It verifies that the domain in SPF/DKIM **aligns** with the `From:` address
+4. It consults the DMARC record (in DNS) to decide what to do:
+   - Accept the email
+   - Mark it as spam (quarantine)
+   - Reject it entirely
+
+---
+
+## 🧾 Example DMARC Record
+
+```dns
```

Comment:
this throws this warning "The language 'dns' is not loaded, falling back to 'txt' for syntax highlighting."